### DISCUSSION TITLE
配置问题请教

### DISCUSSION THREAD

**Discussion by PanZiwei** [wangchucheng/hugo-eureka#49] [Q&A]
你好，
主页配置时有几个问题想要请教：

1. 如何在日志中自动生成如下目录？需要在Configure.toml中配置还是要在post/_index.md中添加呢？具体怎么操作？
以你的博文举例：https://www.wangchucheng.com/zh/posts/migration-from-nginx-to-envoy/

2. 我在archetypes/default.md设置了如下默认参数。
使用hugo new post.md时会自动填充date和lastmod为同一时间，如果我后续修改日志，如何自动更新lastmod？

+++
title = "{{ replace .Name "-" " " | title }}"
date = "{{ .Date }}"
lastmod = "{{ .Date }}"
categories = [ ]
tags = [ ]
slug = ""
draft = false
+++

3. 我自己对Hugo主题开发比较感兴趣，请问你是否有推荐的博客/教程？

谢谢解答！

**Comment by wangchucheng** [comment]
> 1. 如何在日志中自动生成如下目录？需要在Configure.toml中配置还是要在post/_index.md中添加呢？具体怎么操作？

只需要在文章的Front Matter中设置`toc: true`即可。

> 2\. 如何自动更新lastmod？

Front Matter中的值只能手动进行修改，但是你可以通过在config.yaml中设置enableGitInfo: true即可使用Git记录自动更新生成的html中的相关时间。

> 3. 我自己对Hugo主题开发比较感兴趣，请问你是否有推荐的博客/教程？

据我了解目前的教程较少，Hugo的主题开发主要基于基础的前端知识及Hugo的官方文档。
